migrate/20231103_private_subnet_fws.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:firewall_type, %w[ingress egress])
+    create_table(:firewall_rule) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :type, :firewall_type, null: false, default: "ingress"
+      column :start_ip4, :inet, null: false
+      column :end_ip4, :inet, null: false
```

Comment:
not sure if these columns make sense yet. I'll probably only start with a single ip and then make it an ip range